tests/baselines/reference/user/create-react-app.log

```diff
@@ -1,10 +1,147 @@
 Exit Code: 1
 Standard output:
-test/fixtures/issue-5176-flow-class-properties/src/App.js(5,8): error TS8010: 'types' can only be used in a .ts file.
-test/fixtures/issue-5176-flow-class-properties/src/App.js(5,13): error TS1005: ';' expected.
-test/fixtures/webpack-message-formatting/src/AppBabel.js(6,8): error TS17008: JSX element 'div' has no corresponding closing tag.
-test/fixtures/webpack-message-formatting/src/AppBabel.js(8,7): error TS17002: Expected corresponding JSX closing tag for 'span'.
-test/fixtures/webpack-message-formatting/src/AppBabel.js(14,1): error TS1005: '</' expected.
+node_modules/commander/typings/index.d.ts(26,25): error TS2304: Cannot find name 'NodeJS'.
+packages/babel-preset-react-app/create.js(24,5): error TS2591: Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig.
```

Comment:
something went badly wrong with the configuration here, it looks like.

Reply:
This may have just been a bad publish on their part, since I can't seem to repro this locally anymore. I can rerun the user suite?

Reply:
Oh wait, it's a submodule, i probably need to update~

Reply:
Yeah, ok, this is strange, we definitely have `"types": ["node"]` in the config.